.github/workflows/test_models.yml

```diff
@@ -0,0 +1,90 @@
+name: Test Turbine Models
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  test-turbine-models-cpu:
+    runs-on: ubuntu-22.04-64core
```

Comment:
As noted in your comment, let's not do this. This costs $$ to GitHub and would be better to use one of our self managed instances.